compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp

```diff
@@ -165,8 +165,10 @@ void AMDAIETileAndFusePass::runOnOperation() {
     // loops, and the first level of tiling is always using scf.forall and
     // mapped to blocks. Currently we are not using mapping attributes for
     // Conv2d ops, because there could be four parallel tiling dimensions.
+    // Somehow `linalg::isaConvolutionOpInterface()` doesn't work properly.
```

Comment:
Ouch.

Reply:
Can you elaborate, have you tried 

```
if (!isa<linalg::ConvolutionOpInterface>(consumerOp.getOperation()))
```

?

Reply:
Yes, I tried it, and somehow `isa<linalg::ConvolutionOpInterface>` returns true for matmul ops. I think something is broken in upstream, but haven't got time to debug.